glossary.html

```diff
@@ -13,10 +12,7 @@
     <link href="https://fonts.googleapis.com/css?family=Domine&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href ="normalize.css">
-    <link rel="stylesheet" href="main.css">
-=======
     <link rel="stylesheet" href="styles/main.css">
->>>>>>> refs/heads/master
```

Comment:
These markers are git merge conflict markers.  You should never commit them back to git.  Instead, it's up to you to pick which lines to keep and which lines to throw away.  The lines from <<<<<< HEAD to ====== are the lines in your repo and the lines from ======= to >>>>>> refs/heads/master are the lines from the repo you pulled from.  Sometimes you can just pick one side or the other, but sometimes you have to pick and choose lines from both sides.